src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -121,9 +171,14 @@ public ObservableList<Job> getJobList() {
         return jobs.asUnmodifiableObservableList();
     }
 
+    @Override
+    public ObservableList<Company> getCompanyList() {
+        return companies.asUnmodifiableObservableList();
+    }
+
     @Override
     public int hashCode() {
-        return persons.hashCode();
+        return persons.hashCode(); // TODO: this needs to be changed to include all lists
```

Comment:
This seems like a similar thing to the `equals`. I can't find an instance where `hashCode()` is used.